sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -421,7 +421,9 @@
 
   /**
    * @since 4.0
+   * @deprecated no more used since 6.3. See https://jira.sonarsource.com/browse/SONAR-8610
    */
+  @Deprecated
   String HOURS_IN_DAY = "sonar.technicalDebt.hoursInDay";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)